### DISCUSSION TITLE
BT-67 Type does not appear in notice PDF

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#720] [Q&A] [answered]
Hello,

If setting the BT-67 fields like:

` 
 <cac:TenderingTerms>
      <cac:TendererQualificationRequest>
         <cac:SpecificTendererRequirement>
            <cbc:TendererRequirementTypeCode listName="exclusion-ground">bankruptcy</cbc:TendererRequirementTypeCode>
            <cbc:Description languageID="DEU">Code für Bancruptcy</cbc:Description>
         </cac:SpecificTendererRequirement>
         <cac:SpecificTendererRequirement>
            <cbc:TendererRequirementTypeCode listName="exclusion-ground">bankr-nat</cbc:TendererRequirementTypeCode>
            <cbc:Description languageID="DEU">Code für national Bancruptcy</cbc:Description>
         </cac:SpecificTendererRequirement>
      </cac:TendererQualificationRequest>
   </cac:TenderingTerms>
`
only the Description Code in Field BT-67(b) appears in the PDF and not the BT-67(a) Field, but both should be showen.

![image](https://github.com/OP-TED/eForms-SDK/assets/811668/0a784ab5-cae9-4867-8571-9a25cb82fbe0)

What is wrong here?
Best Günter

**Comment by pdonohoe** [comment, marked as answer]
Dear @lepo,
This is a bug. It has been fixed in the recent release of SDK 1.9. The fix will be applied to previous versions of the SDK in the near future.

**Reply by lepo** [reply]
Can you please tell a date when this bug is fixed for sdk 1.8. thank you

**Reply by pdonohoe** [reply]
I'm sorry, I can't give you a date, but I will inform you here when it has been fixed for sdk 1.8.
